pages/documentation/pythnet-price-feeds/sui.mdx

```diff
@@ -31,7 +31,7 @@ yarn add @pythnetwork/pyth-sui-js
 Pyth stores prices off-chain to minimize gas fees, which allows us to offer a wider selection of products and faster update times.
 See [On-Demand Updates](https://docs.pyth.network/documentation/pythnet-price-feeds/on-demand) for more information about this approach.
 Typically, to use Pyth prices on chain,
-they must be fetched from an off-chain price service. The `SuiPriceServiceConnection` class can be used to interact with these services,
+they must be fetched from an off-chain [Hermes](hermes) instance. The `SuiPriceServiceConnection` class can be used to interact with these services,
```

Comment:
off-chain is confusing in this sentence. I think "Hermes web API" would be clearer ? (i realize you didn't write that part)